### PR TITLE
Diffpy.utils

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,4 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not modify this file.

File: recipes/diffpy-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    PDFgui program. The package also includes an interpolation function based on
+    the Whittaker-Shannon formula that can be used to resample a PDF or other
+    profile function over a new grid.
+  doc_url: http://www.diffpy.org/diffpy.utils/
+  dev_url: https://github.com/diffpy/diffpy.utils
+
+extra:
+  recipe-maintainers:
+    - CJ-Wright
+    - pavoljuhas
```
Comment (synapticarbors) [inline recipes/diffpy-utils/meta.yaml]: @pavoljuhas Are you ok being added as a co-maintainer fo the `diffpy.utils` recipe on conda-forge?

Comment (pavoljuhas) [inline recipes/diffpy-utils/meta.yaml]: @synapticarbors - yes, please go ahead.

**Comment by pavoljuhas** [comment]
@CJ-Wright - please add this run_test.py file - https://github.com/diffpy/diffpy.utils/blob/master/conda-recipe/run_test.py

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**